src/server/plugins/engine/services/formsService.js

```diff
@@ -0,0 +1,39 @@
+import { formMetadataSchema } from '@defra/forms-model'
+
+import config from '~/src/server/config.js'
+import { getJson } from '~/src/server/plugins/engine/services/httpService.js'
```

Comment:
I get what you mean about the types in this one. Does anything else use it?

Probably best sticking with one and I'll look at the types in another PR

```suggestion
import { getJson } from '~/src/server/services/httpService.js'
```

Reply:
I'm going to stick with `~/src/server/plugins/engine/services/httpService.js` for now. I'm not sure why there are two `fetch` utils but there's subtly different behaviour between the two, namely this part:

```
    if (res.statusCode !== 200) {
      const err = payload || new Error('Unknown error')
      throw err
    }
```

Let's look at bringing over (or better still, making a shared library) the `URL` ones but typed against wrecks' shortcut methods `get`, `post` etc. This can be done in the next PR for the [clean up ticket](https://dev.azure.com/defragovuk/DEFRA-CDP/_workitems/edit/376156).

Reply:
If you're sure?

I've fixed the types and resolved that subtle difference in:

* https://github.com/DEFRA/forms-runner/pull/95